charts/risingwave/values.yaml

```
@@ -495,6 +495,11 @@ stateStore:
     ##
     bucket: ""
 
+    ## @param stateStore.s3.isForcePathStyle S3 path format. 
+    ## env 'RW_IS_FORCE_PATH_STYLE' is used
+    ##
+    isForcePathStyle: false
```

nit: prefer `forcePathStyle`
```suggestion
    forcePathStyle: false
```